CHANGELOG.next.asciidoc

```diff
@@ -132,6 +132,9 @@ Setting environmental variable ELASTIC_NETINFO:false in Elastic Agent pod will d
 
 *Auditbeat*
 
+- Add `ignore_errors` option to audit module. {issue}15768[15768] {pull}36851[36851]
+- Fix copy arguments for strict aligned architectures. {pull}36976[36976]
```

Comment:
```suggestion
```
I think the first two are not part of this PR, right?

Reply:
Good catch! That probably snuck in in one of my merges